apps/block_scout_web/lib/block_scout_web/views/api/rpc/epoch_view.ex

```diff
@@ -0,0 +1,115 @@
+defmodule BlockScoutWeb.API.RPC.EpochView do
+  use BlockScoutWeb, :view
+
+  alias BlockScoutWeb.API.RPC.RPCView
+
+  alias Explorer.Chain.Wei
+
+  def render(json, %{rewards: rewards})
+      when json in ~w(getvoterrewards.json getvalidatorrewards.json getgrouprewards.json) do
+    RPCView.render("show.json", data: prepare_response(json, rewards))
+  end
+
+  def render(json, %{epoch: epoch})
+      when json in ~w(getepoch.json) do
+    RPCView.render("show.json", data: prepare_response(json, epoch))
+  end
+
+  def render("error.json", assigns) do
+    RPCView.render("error.json", assigns)
+  end
+
+  def prepare_response("getvoterrewards.json", rewards),
+    do: rewards |> wrap_rewards([{"groupAddress", :associated_account_hash}], :celo)
+
+  def prepare_response("getvalidatorrewards.json", rewards),
+    do: rewards |> wrap_rewards([{"groupAddress", :associated_account_hash}], :cusd)
+
+  def prepare_response("getgrouprewards.json", rewards),
+    do: rewards |> wrap_rewards([{"validatorAddress", :associated_account_hash}], :cusd)
+
+  def prepare_response("getepoch.json", nil),
+    do: nil
+
+  def prepare_response("getepoch.json", epoch),
+    do: %{
+      "blockNumber" => to_string(epoch.block_number),
```

Comment:
why do we return everything as strings? numbers are valid json also

Reply:
in the original implementation such things were also returned as strings (so are the numbers, but those can be very big, so it makes sense to pass them around as strings) and I think that's the only reason behind this